content/tutorials/mqtt-snake.textile

```diff
@@ -18,7 +20,9 @@ languages:
   - python
 ably_product: adapters
 tags:
-  - ably-features
+    - MQTT
+    - Multiplayer game
+    - Protocol adapter
```

Comment:
Can we please replace all occurrences of the tag name 'Protocol adapter' with 'Protocol Adapter'?

Reply:
Done (x3)